sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -1034,4 +1034,11 @@ class SQLQuerySuite extends QueryTest with BeforeAndAfterAll {
     rdd.registerTempTable("distinctData")
     checkAnswer(sql("SELECT COUNT(DISTINCT key,value) FROM distinctData"), Row(2))
   }
+
+  test("describe table") {
+    checkAnswer(sql("DESCRIBE EXTENDED testData"),Seq(
```

Comment:
EXTENDED ? 
This seems no different with describe.
I think we'd better do this later after the discusstion.